openedx/core/lib/block_structure/manager.py

```diff
@@ -110,4 +111,5 @@ def clear(self):
         Removes cached data for the block structure associated with the given
         root block key.
         """
+        print "clear() in manager.py ***************"
```

Comment:
Definitely - this is how I'm keeping track of which code is running where while running all my devstack processes. Not intended to be included in final merge by any stretch.